README.md

```diff
@@ -111,6 +111,14 @@ standard-version --no-verify
 
 If you have your GPG key set up, add the `--sign` or `-s` flag to your `standard-version` command.
 
+### Committing generated artifacts in the release commit
+
+If you want to commit generated artifacts in the release commit (e.g. [#96](https://github.com/conventional-changelog/standard-version/issues/96)), you can use the `--commit-all` or `-a` flag. You will need to stage the artifacts you want to commit, so your `release` command could look like this:
+
+```json
```

Comment:
I'm with @bcoe I would add the `prerelease` script for context. Otherwise this looks good. IMHO it's better to give that one more line of context when people are not that likely to follow a link to a lengthy conversation.